app/src/main/java/com/example/plantsapp/presentation/ui/utils/TaskExt.kt

```diff
@@ -16,7 +16,16 @@ fun Task.getIconRes(): Int {
     return when (this) {
         is Task.WateringTask -> R.drawable.ic_watering
         is Task.SprayingTask -> R.drawable.ic_spraying
-        is Task.LooseningTask -> R.drawable.ic_watering
+        is Task.LooseningTask -> R.drawable.ic_loosening
         is Task.TakingPhotoTask -> R.drawable.ic_taking_photo
     }
 }
+
+fun Task.getColorRes(): Int {
+    return when (this) {
+        is Task.WateringTask -> R.color.blue
+        is Task.SprayingTask -> R.color.purple
+        is Task.LooseningTask -> R.color.brown
+        is Task.TakingPhotoTask -> R.color.green_blue
+    }
+}
```

Comment:
If it is used only once , maybe leave it as private inside of ViewHolder you use, if not - it is okay

Reply:
Planned to use this in next pull request on Plant Detail Screen UI improvement